core/server/master/src/main/java/alluxio/master/AlluxioMasterRestServiceHandler.java

```diff
@@ -554,4 +562,23 @@ private Capacity getUfsCapacityInternal() throws IOException {
     return new Capacity().setTotal(mUfs.getSpace(mUfsRoot, UnderFileSystem.SpaceType.SPACE_TOTAL))
         .setUsed(mUfs.getSpace(mUfsRoot, UnderFileSystem.SpaceType.SPACE_USED));
   }
+
+  /**
+   * @summary set the Alluxio log information
+   * @param logName the log's name
+   * @param level the log level
+   * @return the response object
+   */
+  @POST
+  @Path(LOG_LEVEL)
+  @ReturnType("alluxio.wire.AlluxioMasterInfo")
+  public Response logLevel(@QueryParam(LOG_ARGUMENT_NAME) final String logName, @QueryParam
```

Comment:
I let the name put validation delay to the logUtils's setLevel method. is it suitable?